packages/espresso/README.md

```diff
@@ -0,0 +1,115 @@
+
+    // AndroidJUnitRunner and JUnit Rules
+    androidTestImplementation 'androidx.test:runner:1.1.0'
+    androidTestImplementation 'androidx.test:rules:1.1.0'
+
+    // Assertions
+    androidTestImplementation 'androidx.test.ext:junit:1.0.0'
+    androidTestImplementation 'androidx.test.ext:truth:1.0.0'
+    androidTestImplementation 'com.google.truth:truth:0.42'
+
+    // Espresso dependencies
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.1.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-intents:3.1.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-accessibility:3.1.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-web:3.1.0'
+    androidTestImplementation 'androidx.test.espresso.idling:idling-concurrent:3.1.0'
+
+    // The following Espresso dependency can be either "implementation"
+    // or "androidTestImplementation", depending on whether you want the
+    // dependency to appear on your APK's compile classpath or the test APK
+    // classpath.
+    androidTestImplementation 'androidx.test.espresso:espresso-idling-resource:3.1.0'
+}
+```
+
+Create an `android/app/src/androidTest` folder and put a test file in a package-appropriate subfolder, e.g. `android/app/src/androidTest/java/com/example/MainActivityTest.java`:
+
+```
+package com.example.espresso_example;
+
+import static androidx.test.espresso.flutter.EspressoFlutter.onFlutterWidget;
+import static androidx.test.espresso.flutter.action.FlutterActions.click;
+import static androidx.test.espresso.flutter.action.FlutterActions.syntheticClick;
+import static androidx.test.espresso.flutter.assertion.FlutterAssertions.matches;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.isDescendantOf;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withText;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withTooltip;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withType;
+import static androidx.test.espresso.flutter.matcher.FlutterMatchers.withValueKey;
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.Assert.fail;
+
+import androidx.test.core.app.ActivityScenario;
+import androidx.test.espresso.flutter.EspressoFlutter.WidgetInteraction;
+import androidx.test.espresso.flutter.assertion.FlutterAssertions;
+import androidx.test.espresso.flutter.matcher.FlutterMatchers;
+import androidx.test.ext.junit.runners.AndroidJUnit4;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/** Unit tests for {@link EspressoFlutter}. */
+@RunWith(AndroidJUnit4.class)
+public class MainActivityTest {
+
+    @Before
+    public void setUp() throws Exception {
+        ActivityScenario.launch(MainActivity.class);
+    }
+
+    @Test
+    public void performSyntheticClick() {
```

Comment:
We might want to encourage users to use the #click() API instead of #syntheticClick(), as the #click() action issues the click event via the Android system, so it's a more end-to-end test on the app.
I believe we have an example for the #click() action in the tests. Maybe replace with that one in this sample code?

Reply:
I've updated the example to use `click()`